src/doc/dev_manual/pr_create.rst

```diff
@@ -0,0 +1,152 @@
+Creating a Pull Request
+=======================
+
+Overview
+--------
+
+Pull Requests (PRs) allow developers to review work before merging it into the
+develop branch. PRs are extremely useful for preventing bugs, enforcing coding
+practices, and ensuring changes are consistent with VisIt_'s overall architecture. 
+Because PR reviews can take time, we have adopted policies to help tailor the
+review effort and balance the load among developers. We hope these policies will
+help ensure PR reviews are completed in a timely manner. The benefits of reviews
+outweigh the added time.  
+
```

Comment:
Note to self: Add a subsection here with images and instructions about how to submit a PR from a forked repo.

Reply:
In the [RTD generation of this documentation ](https://visit-sphinx-github-user-manual.readthedocs.io/en/task-rusu1-visit-3362-update-pr-template/), I am seeing the above *Overview* text duplicated in both the *Creating a Pull Request* and *Reviewing a Pull Request*. Maybe pull that out to a top-level section entitled *About Pull Requests* with sub-sections for *Creating* and *Reviewing*.

Reply:
I looked into that, and it messed up the menu items.
![Screen Shot 2019-04-23 at 8 13 08 AM](https://user-images.githubusercontent.com/18565891/56592562-c755b880-659f-11e9-8862-cf5b40b2b9af.png)